lms/static/js/instructor_dashboard/send_email.js

```diff
@@ -136,6 +136,9 @@
                     url: url,
                     success: function(data) {
                         if (data.tasks.length) {
+                            sendemail.$history_request_response_error.css({
+                                display: 'none'
+                            });
```

Comment:
Can we delete the error messages instead of hiding this? and what happens where there is an error following this success scenario?

Reply:
if we delete the error message then if any error occurs later on we will have to construct the whole div and then show it
regarding 2nd question I have handled this case as well